steps/parameterList.ts

```diff
@@ -4,4 +4,12 @@ export const parameterDescriptions = {
   withoutLanguageVariants: 'Exclude languages with Country variant. For example `en_CA` will not be present in zim with this argument.',
   mulOnly: 'Skip ZIM files for individual languages',
   createMul: 'Create a ZIM file with all languages',
+  output: 'Output ZIM files in a specific directory',
+}
+
+export const applyParameterConstraints = (argv): boolean => {
+  if (argv.output && Array.isArray(argv.output)) {
```

Comment:
do we really need to check for that, since you've specified it's a string?

Reply:
Yes we do, otherwise multiple values are automatically concatenated with a comma ... not what we want